styles/sass/lib/buildingblocks/_card.scss

```diff
@@ -23,6 +23,16 @@
             font-size: 58px;
         }
     }
+    .mx-link {
```

Comment:
Maybe make a nice helper class (and design property) for ellipsis and add them to the building block in the model. (you also have to update all used instances then.